### DISCUSSION TITLE
Each Sender its own Signature

### DISCUSSION THREAD

**Discussion by arentsen** [tutao/tutanota#5234] [Ideas]
It would be great if each Sender could have its own Signature.

**Comment by serrq** [comment]
You mean cryptographic signature?

**Comment by arentsen** [comment]
I have a Tutanota email address and one linked to another external domain and I would like to be able to connect different signatures to each email address.

**Comment by xrazis** [comment]
This will be really useful. I use both a personal and a business email under the same account and I have to erase my personal signature and then append the company one.

**Comment by hrb-hub** [comment]
We agree that this is quite useful, and we do plan on implementing it sometime in the not so distant future :)
